### DISCUSSION TITLE
TinyMCE extended_valid_elements

### DISCUSSION THREAD

**Discussion by richard-enov** [PiranhaCMS/piranha.core#1474] [Q&A]
Hi

I need to add the TinyMCE extended_valid_elements. I can see that this was added in version 8.3, issue #1154 but it doesnt seem to be working for me.  Currently running version 8.4 and I'm not sure what else needs to be done apart from editing editorconfig.json. I've verified that the option is not loading in the JS using the browser dev tools.

Any help is appriciated.

Thanks

**Comment by tidyui** [comment]
TinyMCE is a **killer** at caching resources. Did you make sure to clear the cache and reload all of the resources in your browser. I've used the TinyMCE editor since 2006 and it has **always** had these "issues" :)
